src/test/resources/TestNG.xml

```diff
@@ -2,12 +2,13 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="TestNG">
-
+    <parameter name="BaseURL" value="https://bbb.testpro.io/"/>
     <test name="Example Test" preserve-order="false">
         <classes>
-            <class name="Homework18">
+            <class name ="BaseTest"/>
+            <class name="Homework19">
                 <methods>
-                    <include name="playSong"/>
+                    <include name="deletePlaylist"/>
                 </methods>
 
             </class>
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223073641-8825693a-7261-4037-8ec0-845ebf4bacfa.png)

Reply:
I fixed the problem by adding the code to maximize the browser. "X PLAYLIST" is not clickable when the browser is small. 
How do I update it to Git? I clicked "push" but am not able to create pull request. It's still confusing to me.